docs/getting-started/raster/raster_must_know_terms.md

```diff
@@ -0,0 +1,44 @@
+
+Term | Description
+:--- | :---
+**Raster Geodetic Datum**  | World Geodetic System of WGS 84 (also known as WGS 1984 ensemble: EPSG:4326 for 2D coordinate reference system (CRS))
+**Raster Map projection**  | Latitude / Longitude Projection
+**Raster Best Practice Protocol**  | WMTS
+**Raster Tiling scheme**  | InspireCRS84Quad (World CRS84 TileMatrixSet), That tiling schemes consists of two 256x256 tiles at its zoom level 0, in EPSG:4326 CRS, with extent in longitude in the range [-180,180] and in latitude in the range [-90,90].
+**Raster Tile Size**  | 256*256
+**Raster Tile Format**  | PNG
+**Auth token** | A JWT token that needs to be supplied by raster PO (product team) and be supplied to raster services via QUERYPARAM, there are different kinds of token permissions (raster serving CSW, WMTS, WMS, export, etc.) 
```

Comment:
A JWT token (you can aquire it by contacting our product owner) for raster services passed via http `QUERY_PARAM`. There are different kinds of token permissions (raster serving CSW, WMTS, WMS, export, etc.)

Reply:
fixed